pkgs/development/tools/swc/default.nix

```diff
@@ -15,12 +15,10 @@ rustPlatform.buildRustPackage rec {
   src = fetchCrate {
     pname = "swc_cli";
     inherit version;
-    hash = "sha256-8zbxE1qkEWeSYt2L5PElZeJPRuK4Yiooy8xDmCD/PYw=";
+    hash = "sha256-6n6zHMV87h1kmjzEmdE86/toHI99q2HO1EEGHUE9sg8=";
   };
 
-  cargoHash = "sha256-kRsRUOvDMRci3bN5NfhiLCWojNkSuLz3K4BfKfGYc7g=";
-
-  buildFeatures = [ "swc_core/plugin_transform_host_native" ];
```

Comment:
Package no longer builds with this. I traced it back to some Cargo.toml changes [here](https://github.com/swc-project/swc/commit/f717cf21cc1cea5e30e87c4d08861daadb25ab14#diff-4de2b397cbfaf8a35cc364b6a40e18997477ae769f731aeb3b710ec7761b504fR17), although I'm not 100% sure about the root cause. But the build succeeds now with the default features.